### PR TITLE
Re-include command line instructions to build apk

### PR DESCRIPTION
**Pull request by Irineu333**
### Description

As @akash07k and @PatrykMis explained to me, it is easier for a visually impaired person to develop for Android, using editors like [Notepad++](https://github.com/notepad-plus-plus/notepad-plus-plus) or [VSCodium](https://github.com/VSCodium/vscodium), instead of Android Studio or Intellij, widely used by sighted people. These IDEs, unfortunately, still lack in terms of accessibility.

With this in mind, it is essential that the readme has instructions on how to build via the command line to make it easier for curious people to get started.

### Improvements
- Adde instructions on how to build with command line in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
 ## Build
 In development, choose the *debug build type*, because for the *release build type*, it's necessary to create a [keystore](https://developer.android.com/training/articles/keystore) and specify its properties in the `keystore.properties` file (use `keystore.properties.example` as a base).
 
-### IDE
-To build you need [Android Studio](https://developer.android.com/studio/releases) (Flamingo or later).
```
Comment (PatrykMis) [inline README.md:8]: Why you've deleted this? It's still true; we can eventually write that the latest version is required as we use modern build infrastructure and trying to follow latest stable versions of dependencies like AGP. In older versions of Android Studio the project may not work correctly or at all.

File: README.md [hunk 2]
```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
...
+### Android Studio
 
-1. Clone the repository to your local machine.
-2. Open the project in the IDE.
-3. Run the **app** module.
+1. Download and install [Android Studio](https://developer.android.com/studio).
```
Comment (PatrykMis) [inline README.md]: Regarding previous comment, we can drop previous note and write `1. Download and install [the latest version of Android Studio](https://developer.android.com/studio).`

File: README.md [hunk 3]
```diff
@@ -4,12 +4,28 @@ A open source screen reader for android.
...
+
+### IntelliJ IDEA
+
+1. Download and install [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. Clone the repository to your local machine.
+3. Open the project in the IntelliJ IDEA.
+4. Run the **app** module.
+
+### Command line
+1. Download and install [JDK 17](https://www.oracle.com/java/technologies/downloads/#java17) or newer.
```
Comment (PatrykMis) [inline README.md]: I would prefer Adopt OpenJDK (`1. Download and install [Adoptium OpenJDK 17](https://adoptium.net/download/) or newer.`).

OpenJDK is free software and has binaries for all major systems and distributions and is available from all known package managers.

**Review by PatrykMis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Irineu333) [reply README.md:8]: Ok

File: README.md (hunk 3 above)
Comment (Irineu333) [reply README.md]: ok

**Review by PatrykMis — APPROVED** [review]

**Pull request merged**